src/components/learner-credit-management/invite-modal/InviteModalSummary.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Card, Stack } from '@edx/paragon';
+
+import InviteModalSummaryEmptyState from './InviteModalSummaryEmptyState';
+import InviteModalSummaryLearnerList from './InviteModalSummaryLearnerList';
+import InviteModalSummaryErrorState from './InviteModalSummaryErrorState';
+
+const InviteModalSummaryContents = ({
+  hasLearnerEmails,
+  learnerEmails,
+  hasInputValidationError,
+}) => {
+  if (hasLearnerEmails) {
+    return (
+      <InviteModalSummaryLearnerList
+        learnerEmails={learnerEmails}
+      />
+    );
+  }
+  if (hasInputValidationError) {
+    return <InviteModalSummaryErrorState />;
+  }
+  return <InviteModalSummaryEmptyState />;
+};
+
+const InviteModalSummary = ({
+  memberInviteMetadata,
+}) => {
+  const {
+    isValidInput,
+    lowerCasedEmails,
+  } = memberInviteMetadata;
+  const hasLearnerEmails = lowerCasedEmails?.length > 0 && isValidInput;
```

Comment:
nit: Is `lowerCasedEmails` ever _not_ an array? It appears it's created as an empty array, which seems to imply it would always exist, and not need the `?.`.